package.json

```diff
@@ -72,6 +72,7 @@
     "@next/polyfill-module": "workspace:*",
     "@next/polyfill-nomodule": "workspace:*",
     "@next/swc": "workspace:*",
+    "@opentelemetry/api": "1.4.1",
```

Comment:
can you run `./scripts/check-pre-compiled.sh` ?

Reply:
It told me `Detected changes`, finished but it didn't change anything.